video/out/opengl/common.c

```diff
@@ -579,8 +579,6 @@ void mpgl_load_functions2(GL *gl, void *(*get_fn)(void *ctx, const char *n),
         int glsl_major = 0, glsl_minor = 0;
         if (shader && sscanf(shader, "%d.%d", &glsl_major, &glsl_minor) == 2)
             gl->glsl_version = glsl_major * 100 + glsl_minor;
-        // GLSL 400 defines "sample" as keyword - breaks custom shaders.
-        gl->glsl_version = MPMIN(gl->glsl_version, 330);
```

Comment:
I think this should still limit to 400, with the comment removed. It's better not to allow an unbounded GLSL version, since they might add new conflicting keywords.